content/docs/guides/neon-identity.md

```diff
@@ -0,0 +1,149 @@
+---
+title: About Neon Identity
+subtitle: Automatically sync user profiles from your auth provider directly to your database
+enableTableOfContents: true
+---
+
+_<b>Q:</b> This EAP message is re-purposed from another feature. We can modify if, for example, we want to link to sign up for the EAP etc_
+
+<ComingSoon />
+
+**Neon Identity** connects your authentication provider to your Neon database, automatically synchronizing user profiles. Access your user data directly in your database environment, with no custom integration code needed.
+
+## Authentication and synchronization
+
+When implementing user authentication, it is common to use dedicated authentication providers like StackAuth, Clerk, and others to handle the complexities of user identity, passwords, and security. However, keeping your database in sync with these providers typically requires additional development work.
+
+Neon Identity solves this by maintaining a direct connection between your auth provider and database, ensuring your application always has access to up-to-date user information right from your database.
+
+_<b>Q:</b> What specific benefits can we list out here? Less API calls, etc?_
+
+## How Neon Identity works
+
+When you set up Neon Identity, we create a `neon_identity` schema in your database. As users authenticate and manage their profiles in your auth provider, their data is automatically synchronized to your database.
+
+Here is the basic flow:
+
+1. **User profiles are created and managed in your authentication provider**
+
+   This view shows the list of users inside your auth provider (e.g. StackAuth). When new users sign up or update their profiles in your app, their data first appears here:
+   ![Users in Auth Provider](/docs/guides/stackauth_users.png)
+
+2. **Neon Identity syncs their data to your database**
+
+   This view shows the synced user profiles in Neon Identity. This is where Neon manages the connection between your database and the authentication provider.
+   ![Same users in Neon Identity](/docs/guides/identity_users.png)
+
+3. **The data is immediately available in your database**
+
+   The synchronized data is immediately available in the `neon_identity.users_sync` table. Here's an example query to inspect the synchronized data:
```

Comment:
Should we have an asterisk saying this could take a few seconds in reality?

Reply:
Actually, we are doing this process `synchronously`, meaning that the data should be available right after the OAuth flow is over

Reply:
Ok, clarified the timing to a generic "shortly after auth provider processes changes"